src/library.js

```diff
@@ -612,6 +616,30 @@ const createPilingJs = (rootElement, initOptions = {}) => {
       );
   };
 
+  const updatePileItemStyle = pile => {
+    const { itemOpacity } = store.getState();
+
+    pile.items.forEach((item, i) => {
+      item.alpha = isFunction(itemOpacity)
```

Comment:
`item` is not the actual item instance but the raw PIXI sprite object. This would generally be okay but as discussed on Slack this is not ideal as we're exposing `item` to the user in the form of the first argument in the next line.

Unfortunately, this requires a bit more refactoring in the way that we pass the item instances to a `pile` instance. The item should then have an opacity function very similar to the one that I added to the `pile` instances.